admin

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python2
+"""
+    SecureDrop Admin Toolkit.
+    
+    For use by administrators to install, maintain, and manage their SD
+    instances.
+"""
+
+
+import argparse
+import subprocess
+import os
+import sys
+import getpass
+
+ANSIBLE_PATH = "./install_files/ansible-base"
+SITE_CONFIG = "group_vars/all/site-specific"
+VENV_DIR = ".venv"
+
+def error(msg, level='warning', exit=True):
+    """ Stupid wrapper function to normalize warnings """
+    print("{} - {}".format(level.upper(), msg))
+    if exit:
+        sys.exit(1)
+
+def activate_venv():
+    """Use to activate the local virtualenv"""
+    activate = os.path.join(VENV_DIR, 'bin/activate_this.py')
+    try:
+        exec(open(activate).read())
```

Comment:
`exec` makes me queasy, but I guess it's your only option in this case. I slightly prefer `execfile` as suggested in http://jelly.codes/articles/python-virtualenv-from-within-python/, but they're functionally equivalent.